content/chat/replies.textile

```diff
@@ -0,0 +1,122 @@
+
+h2(#enable). Enable replies and threads
+
+To use replies and threads, you'll need to enable this feature as a new "channel rule":/channels#rules within your application in the dashboard.
+
+Namespaces enable your chat to be grouped based on a prefix included as part of the channel name. Defining namespaces helps manage channels, such as defining the speed at which members can post messages or whether members can interact with messages such as reactions, replies, or threads.
+
+The namespace for your replies and threads will need to be prefixed within this "channel rule":/channels#rules.
+
+<!-- DASHBOARD-INSTRUCTION: Enable message interactions -->
+To enable message interactions for a channel:
+
+1. Login to your "Ably dashboard":https://ably.com/login.
+2. Select the app you are working with.
+3. Click *Settings* on the right side of the navigation bar.
+4. Scroll down to *Channel rules* and click *Add new rule*.
+5. Give your new rule a Namespace or provide a specific Channel ID. This should be the primary channel, and not the channel you're using for replies.
+6. Select the *Message interactions enabled* checkbox then click *Create channel rule*.
+
+<a href="/images/realtime/message-interactions.png" target="_blank">
+  <img src="/images/realtime/message-interactions.png" style="max-width:80%" alt="Message interactions in dashboard">
+</a>
+
+Once a channel has message interactions enabled, messages received on that channel will contain a unique @timeSerial@ that can be referenced by later messages. These message replies also add a @ref@ object to message payloads that enables you to indicate what an interaction represents, and which message it is referring to. The @ref@ object contains two fields:
+
+- @type@ (String) := a constant representing the reason for the interaction.
+- @timeSerial@ (String) := a unique identifier used to reference a specific message, assigned when messages are sent in message interaction enabled channels. When replying to a message use the timeSerial of the parent message.
+
+h3(#publish). Publish message replies or threads
+
+With message interactions enabled, each message now automatically has the @timeSerial@ attached, enabling the message to be referenced in replies or threads. To reference a previous message, include the @ref@ object inside the @extras@ object of your payload.
+
+The @extras@ object is a JSON object of arbitrary key-value pairs that may contain metadata such as those required when publishing message replies. When referencing messages the @ref@ object needs to contain the following two key-value pairs:
+
+* A @type@ constant string defining the reason for interacting with the message (replying), for message replies it'll be @chat.reply@, or for message threads it could be @chat.thread@. Note that Ably has reserved strings beginning with @com.ably.@.
+* The @timeserial@ string of the message to interact with.
+
+The following is an example of sending a message reply:
```

Comment:
When I read this kind of content I find it useful to see what IDs might look like and where do I get them from. Not  sure if this is general, but here's the question: would it be valuable to add the example of a message that was received with Message Interactions enabled (to show where the time serial is) and then how to reply to that message, using the raw value of that time serial in the example below?